deployment/deployer.yaml

```diff
@@ -67,6 +67,16 @@ items:
       - watch
       - delete
       - update
+  - apiVersion: v1
```

Comment:
The thought was to automate the upgrade process for the EFK stack as much as possible since its fairly involved.

After discussing with @sosiouxme we can get away with deleting the daemonset itself to unschedule the pods since we'll need to do that anyway if we are to make any changes to the daemonset as part of the upgrade.  And we can output instructions for labelling nodes to redeploy Fluentd after the upgrade instead.